src/Circle.java

```diff
@@ -0,0 +1,25 @@
+public class Circle {
+    Point s;
```

Comment:
what is this point and why it is not private?

Reply:
This is the center point of the circle

Reply:
s means center? It is not clear. Change the name.